git.md

```diff
@@ -81,7 +81,27 @@ and [configure Git](#configuring-git).
 
 ## Installation on Windows
 
-Git should be installed on your computer as part of the Bash installation.
+### If you have installed Git Bash
+
+In this case Git should be installed on your computer as part of the Bash installation.
+
+Possible problem with this approach is that Git Bash does not see
+the Python and Anaconda scripts. See also [the troubleshooting page](/installation/troubleshooting/).
+
+Please also [verify your installation](#how-to-verify-the-installation)
+and [configure Git](#configuring-git).
+
+
+### Installation using Conda
+
+This assumes you have already installed Anaconda. Then you can install Git in one line:
+
+```shell
+$ conda install git
+```
```

Comment:
with this approach shouldn't one be using anaconda prompt rather than git-bash?

Reply:
Yes exactly. Should I clarify this better?

Reply:
Or asking differently: I don't see the contradiction/problem. But I can imagine this is not clear enough.

Reply:
i'll look at the whole material for better overview and see if this might be misunderstood, brb

Reply:
ok it's clear that GitBash and conda are two alternative methods to get Git, but maybe clarify by writing "Then you can install Git in one line in the Anaconda Prompt:"

Reply:
Hopefully clearer now.